docs/namespace.md

```diff
@@ -1,15 +1,30 @@
 # Run in custom namespace
 
-In Ark version 0.7.0 and later, you can run Ark in any namespace. To do so, you specify the namespace in the YAML files that configure the Ark server. You then also specify the namespace when you run Ark client commands.
+In Ark version 0.7.0 and later, you can run Ark in any namespace. To do so, you specify the
+namespace in the YAML files that configure the Ark server. You then also specify the namespace when
+you run Ark client commands.
 
 ## Edit the example files
 
-The Ark repository includes [a set of examples][0] that you can use to set up your Ark server. The examples specify only the default `heptio-ark` namespace. To run in another namespace, you edit the relevant files to specify your custom namespace.
+The Ark repository includes [a set of examples][0] that you can use to set up your Ark server. The
+examples place the server in the `heptio-ark-server` namespace, and backup/schedule/restore/config
+data in the `heptio-ark` namespace.
+
+To run the server in another namespace, you edit the relevant files, changing `heptio-ark-server` to
+your desired namespace.
+
+To store your backups, schedules, restores, and config in another namespace, you edit the relevant
+files, changing `heptio-ark` to your desired namespace.
+
+NOTE: it is strongly advised that you run the Ark server in its own namespace, separate from the
+namespace where your backups, schedules, restores, and config live. If you don't, you may not be
```

Comment:
edit:
WARNING: It is recommended to run the Ark server in one namespace, and your backups, schedules, restores, and config in a different namespace. You might encounter issues with deleting a single Ark namespace that runs everything.
(edits for clarity only)